radicle/src/cob/patch.rs

```diff
@@ -394,10 +394,34 @@ impl Revision {
         self.description.get()
     }
 
+    pub fn author(&self) -> &Author {
+        &self.author
+    }
+
+    pub fn base(&self) -> &git::Oid {
+        &self.base
+    }
+
+    pub fn oid(&self) -> &git::Oid {
+        &self.oid
+    }
+
+    pub fn timestamp(&self) -> &Timestamp {
+        &self.timestamp
+    }
+
+    pub fn discussion(&self) -> &Thread {
+        &self.discussion
+    }
+
     pub fn merges(&self) -> impl Iterator<Item = &Merge> {
         self.merges.iter().map(|m| m.get())
     }
 
+    pub fn merges_is_empty(&self) -> bool {
+        self.merges.is_empty()
+    }
```

Comment:
This function is too specific to be useful. If you return `impl ExactSizeIterator` from `merges()`, then you can just use `merges().is_empty()`

Reply:
Ah, perhaps we need to implement that on `LWWSet` then. Want to try?

Reply:
Ok I tried a few times to get this working, but in the end the major point is that `ExactSizeIterator::is_empty()` is still part of the nightly_only API.
https://doc.rust-lang.org/stable/std/iter/trait.ExactSizeIterator.html#method.is_empty

So we need still would need to rely on checking if the length is equal to 0, which doesn't require any new implementation.

Maybe we can revisit this once `ExactSizeIterator::is_empty()` is stable?

Reply:
Yeah I tried, but on one hand the `is_empty()` method on `ExactSizeIterator` is still nightly only and I'm not able to implement `ExactSizeIterator` successfully on `LWWSet`